wkcuber/api/dataset.py

```diff
@@ -442,6 +443,40 @@ def add_symlink_layer(self, foreign_layer_path: Union[str, Path]) -> Layer:
             self.get_layer(layer_name)._setup_mag(resolution.mag.to_layer_name())
         return self.layers[layer_name]
 
+    def add_copy_layer(self, foreign_layer_path: Union[str, Path]) -> Layer:
+        """
+        Copies the data at `foreign_layer_path` which belongs to another dataset to the current dataset.
+        Additionally, the relevant information from the `datasource-properties.json` of the other dataset are copied too.
+        """
+
+        foreign_layer_path = Path(os.path.abspath(foreign_layer_path))
+        layer_name = foreign_layer_path.name
+        if layer_name in self.layers.keys():
+            raise IndexError(
+                f"Cannot copy {foreign_layer_path}. This dataset already has a layer called {layer_name}."
+            )
+
+        shutil.copytree(foreign_layer_path, join(self.path, layer_name))
```

Comment:
Is it enough to copy the data like this, or should it be distributed like https://github.com/scalableminds/voxelytics/blob/master/voxelytics/distributed_copy.py?

Reply:
For my current use case, copytree will do fine :) It may make sense to add parallelization later.